decidim-core/decidim-core.gemspec

```diff
@@ -61,6 +61,7 @@ Gem::Specification.new do |s|
   s.add_dependency "rubyzip", "~> 1.2", ">= 1.2.2"
   s.add_dependency "sassc", "~> 1.12", ">= 1.12.1"
   s.add_dependency "sassc-rails", "~> 1.3"
+  s.add_dependency "seven_zip_ruby", "~> 1.2", ">= 1.2.2"
```

Comment:
It seems like this project hasn't been maintained on the last 3 years and doesn't have much community, could we find an alternative? 
https://github.com/masamitsu-murase/seven_zip_ruby

Reply:
I guess we could. Have you looked into it?

@ankane thought it was good enough when he wrote [this article](https://ankane.org/sensitive-data-rails#downloads) in November 20, 2018, I wonder if he'd choose a different gem these days.

Reply:
@aitorlb it's still the best gem I'm aware of, but let me know if you find something better

Reply:
@ankane thanks for chiming in :+1:

Reply:
@andreslucena can you please give response

Reply:
:+1: go ahead

Reply:
Also great post @ankane !
I really liked it.